pkg/toolchains/rpmbuild.bzl

```diff
@@ -0,0 +1,69 @@
+# Copyright 2020 The Bazel Authors. All rights reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#    http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+"""toolchain to provide an rpmbuild binary."""
+
+RpmbuildInfo = provider(
+    doc = """Platform inde artifact.""",
+    fields = {
+        "name": "The name of the toolchain",
+        "valid": "Is this toolchain valid and usable?",
+        "label": "The path to a target I will build",
+        "path": "The path to a pre-built rpmbuild",
```

Comment:
This won't conflict with names in other toolchains. They are scoped to the RpmbuildInfo provider.
I could have a longish discussion about interfaces to things, but the short answer is that if I have N types, and each has the need for a path to an executable, I like to use the type as the descriiminator and keep the attribute path named that in all instances.

Reply:
That makes sense, but that sounds to me more like a discussion regarding a series of distinct toolchains rather than a single toolchain containing multiple entries, which is what I had in mind.

Are you proposing that there be separate toolchains for other tools that are rpm-related, like `rpm2cpio` and `cpio`?  This isn't necessarily a bad thing, I just wanted to make sure that I understand what is being suggested.

Reply:
Ah..... I see what you mean. I misunderstood.  For usibility, rpm2cpio should go with the rpmbuild toolchain, and thus, the names should be different.  I'll correct this in a followup CL.